src/sasl/cyrsasl_external_verification.erl

```diff
@@ -3,7 +3,11 @@
 %%% @author Denys Gonchar <denys.gonchar@erlang-solutions.com>
 %%% @doc cyrsasl external verification backend
 %%%
-%%% this module is added for demo & testing purposes only.
+%%% This is experimental prototype module for demo & testing purposes only.
+%%% It allows to get a list of certificates from third party Certificate Authority service.
```

Comment:
%%% It allows to get a list of certificates from a third party Certificate Authority service.